include/nbl/asset/IDescriptorSetLayout.h

```diff
@@ -396,9 +406,9 @@ class IDescriptorSetLayout : public virtual core::IReferenceCounted  // TODO: tr
 
 	CBindingRedirect m_descriptorRedirects[static_cast<uint32_t>(asset::IDescriptor::E_TYPE::ET_COUNT)];
 	CBindingRedirect m_immutableSamplerRedirect;
-	CBindingRedirect m_mutableSamplerRedirect;
+	CBindingRedirect m_mutableCombinedSamplerRedirect;
 
-	core::smart_refctd_dynamic_array<core::smart_refctd_ptr<sampler_type>> m_samplers = nullptr;
+	core::smart_refctd_dynamic_array<core::smart_refctd_ptr<sampler_type>> m_immutableSamplers = nullptr;
```

Comment:
how do you know which part of `m_immutableSamplers` is for COMBINED's immutable samplers and for SAMPLER immutables?

Reply:
or do you now need to know ?

Reply:
I don't recall needing to know, I think I just treat all immutable samplers equally

Reply:
yeah, reading the code it looks like you don't need to know, will resolve after I read everything